tools/commit-checker/check.py

```diff
@@ -209,12 +237,18 @@ def main():
             continue
         found = True
         total_problems += 1
+        no_commit_ids.add(issue.issue_id)
         pretty_print_issue(issue, **vars(args))
+        if(issue.issue.fields.resolution):
```

Comment:
Here and below: why not put this into `pretty_print_issue`?

Reply:
done